src/components/Button/native/Button.js

```diff
@@ -44,9 +47,38 @@ const Button = styled.TouchableOpacity`
     `,
   }[p.size])}
   ${(p) => p.block ? 'align-self: stretch;' : 'align-self: flex-start;'}
-  ${(p) => p.disabled ? 'opacity: 0.5;' : ''}
+  ${(p) => p.disabled ? 'opacity: 0.5;' : 'opacity: 1'}
```

Comment:
you can ignore the false statement styling with an empty string
`${(p) => p.disabled ? 'opacity: 0.5;' : ''}`

Reply:
This doesn't work. For example if a button is disabled and I want to change it to the normal color as soon as its enabled, just passing the empty string doesn't work. I have to reset it back to opacity 1, hence added this.

Reply:
I see, must be handled differently on web and native then.